spec/bolt/module_installer/puppetfile_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+require 'bolt/module_installer/puppetfile'
+
+describe Bolt::ModuleInstaller::Puppetfile do
+  let(:path)       { @project + 'Puppetfile' }
+  let(:moduledir)  { @project + '.modules' }
+  let(:name)       { 'puppetlabs/yaml' }
+  let(:version)    { '0.1.0' }
+  let(:mod)        { [double('mod', name: name, version: version, to_spec: "mod '#{name}'")] }
+  let(:puppetfile) { described_class.new(mod) }
+
+  around(:each) do |example|
+    Dir.mktmpdir(nil, Dir.pwd) do |project|
+      @project = Pathname.new(project)
```

Comment:
Can't this use `with_project`?

Reply:
Ya. :) I think this is just left over from before I made `with_project`.